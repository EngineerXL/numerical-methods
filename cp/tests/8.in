16677028992366760668515
6985616526181666899242175
