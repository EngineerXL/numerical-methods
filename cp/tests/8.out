116499329336688035324680315456440707249882620125
